### PR TITLE
Add SHT30 Alternative Packaging products

### PR DESCRIPTION
**Pull request by tyeth**
Adds aliases for the sht30_shell + sht30_weatherproof

### CODE REVIEW

**Comment by tyeth** [comment]
Blocked by I2C Decode Error, sensor name is too long (15character max)

File: src/components/i2c/WipperSnapper_I2C.cpp [hunk 1]
```diff
@@ -441,7 +441,10 @@ bool WipperSnapper_Component_I2C::initI2CDevice(
     _sht4x->configureDriver(msgDeviceInitReq);
     drivers.push_back(_sht4x);
     WS_DEBUG_PRINTLN("SHT4X Initialized Successfully!");
-  } else if (strcmp("sht3x", msgDeviceInitReq->i2c_device_name) == 0) {
+  } else if ((strcmp("sht3x", msgDeviceInitReq->i2c_device_name) == 0) ||
+             (strcmp("sht30_shell", msgDeviceInitReq->i2c_device_name) == 0) ||
+             (strcmp("sht30_weatherproof", msgDeviceInitReq->i2c_device_name) ==
```
Comment (brentru) [inline src/components/i2c/WipperSnapper_I2C.cpp:446]: Could you truncate the name? `strcmp` is pretty quick but a shorter string length may perform better. Comparison btween strings is expensive and we do a lot of that within this i2c base class.

Comment (tyeth) [reply src/components/i2c/WipperSnapper_I2C.cpp:446]: Renamed to sht30_mesh

**Review by brentru — APPROVED** [review]
Looks OK and matches component rename

**Comment by brentru** [comment]
Can merge when ready

**Pull request merged**